Atomic/util.py

```diff
@@ -254,3 +253,19 @@ def urllib3_disable_warnings():
             # Except only call disable-warnings if it exists
             if hasattr(urllib3, 'disable_warnings'):
                 urllib3.disable_warnings()
+
+
+def skopeo(image):
```

Comment:
I mean does this `image` gets fully qualified? if the docker daemon on the host is configured with `--add-registry=myregistry.com:5000` and one does `atomic verify busybox` we should make sure to run `skopeo` with:

`skopeo myregistry.com:5000/busybox` <- if this fails otherwise return result from myregistry.com:5000
`skopeo docker.io/busybox`